.vitepress/theme/components/VueJobs.vue

```diff
@@ -29,35 +44,32 @@ onMounted(async () => {
         target="_blank"
       >
         <div class="vj-company-logo">
-          <img :src="job.organization.avatar" alt="" />
+          <img :src="job.organization.avatar"
+            :alt="`${job.organization.name}'s logo for hiring purposes on VueJobs.com.`" />
```

Comment:
The attributes need to be consistent, either all of the same line or all on different lines. I think I'd also go for a shorter `alt` text:
```suggestion
          <img
            :src="job.organization.avatar"
            :alt="`Logo for ${job.organization.name}`"
          />
```